src/editors/xrWeatherEditor/property_converter_color.cpp

```diff
@@ -83,7 +83,7 @@ bool property_converter_color::CanConvertFrom(ITypeDescriptorContext ^ context,
 Object ^ property_converter_color::ConvertFrom(ITypeDescriptorContext ^ context, CultureInfo ^ culture, Object ^ value)
 {
     String ^ string = dynamic_cast<String ^>(value);
-    if (!value)
```

Comment:
I just checked the code and I see that this replacement is wrong and the code was written like that intentionally.

Reply:
But it makes sense to swap the code like that:
```
	if (!value)
		return (inherited::ConvertFrom(context, culture, value));

	String ^ string = dynamic_cast<String ^>(value);
	try
	{
		...
```

Reply:
Good to catch it before the merge.

I'll move cast below value check than.

Reply:
> But it makes sense to swap the code like that:
> 
> ```
> 	if (!value)
> 		return (inherited::ConvertFrom(context, culture, value));
> 
> 	String ^ string = dynamic_cast<String ^>(value);
> 	try
> 	{
> 		...
> ```

Done, thank you. Hope space doesn't matter here.

Reply:
Well, it doesn't look good, sorry
![trollsad](https://cdn.discordapp.com/emojis/639553094316851200.png?v=1)

Reply:
It'll be better to use ClangFormat or similar tool to make autonate boring fornatting task.

I'll fix formatting tomorrow, thanks!

Reply:
Done!